discover/portal/articles-dxp/140-workflow/02-kaleo-forms/02-assigning-kaleo-forms.markdown

```diff
@@ -0,0 +1,160 @@
+## Assigning Forms for Workflow Tasks [](id=adding-task-forms)
+
+In the fourth (and final) step of the New Process wizard, assign a form to each
+task in the workflow definition (and to the StartNode to get the process kicked
+off). All forms you work with here will be either the entire field set you
+defined in step two, or a subset of it. 
+
++$$$
+
+**Note:** In this step, you can't add any proper form fields (fields that accept
+user input). If you forgot to add a Select field to the field set in step two,
+for example, you cannot add one here. Instead go back and add it to the field
+set.
+
+While you can't add form fields, you can configure the existing fields
+differently for each form, and you can add these elements to the form that
+enhance its usability:
+
+1. Add a *Fieldset* to provide headings in a form. For example, name it
+   after the workflow task to provide clarity to the task assignee.
+2. Use a *Paragraph* field to add a block of text to the form. For example,
+   provide instructions to assignees about what's expected of them to complete
+   the task.
+3. Use the *Separator* to separate sections of the form visually. For example,
+   combine a Paragraph and a Separator to instruct the assignee to focus on a
+   certain section of the form.
+
+![Figure 4: Use Fieldsets, Paragraphs, and Separators to enhance the appearance of your workflow task forms.](../../../images-dxp/kaleo-forms-field-types.png)
+
+$$$
+
+For review, the Spa Order Process has these fields:
+
+- Customer Name
+- Requested Spa Technician
+- Services Requested 
+- Returning Customer?
+- Preferred Date
+- Preferred Time
+- Available Date
+- Available Time
+- Approved
+- Managerial Comments
+
+Not all the available fields should appear on each form. For example, the
+Approved field should only appear on the form for the managerial review task.
+
+Each applicable workflow node (the initial state and each task) appears in a row
+with its associated form and a button that lets you assign a form. In the Spa
+Order Workflow definition, there are four nodes that need forms: StartNode,
+Technician Claim, Managerial Approval, and Final Order. Next you'll assign a 
```

Comment:
The Workflow tasks in the following screenshot have the name slightly different from the names written here. In the screenshot, the names are _Technician_ and _Manager Approval_.